docs/create_broker.md

```diff
@@ -0,0 +1,246 @@
+`alert_id`. Once a query has been performed, any resulting alert may be made into
+a TOM target directly from the query results page.
+
+```python
+@classmethod
+def fetch_alert(clazz, alert_id):
+    response = requests.get(broker_url)
+    test_alerts = response.json()
+    response.raise_for_status()
+    for alert in test_alerts:
+        if (alert['id'] == int(alert_id)):
+            return alert
+    return None
+```
+
+We query the broker, obtain our JSON data, and return the alert whose id
+matches the `alert_id` parameter, so a TOM Alert may be created from it.
+
+#### `to_generic_alert` Class Method
+In order to standardize alerts and display them in a consistent manner,
+the `GenericAlert` class has been defined within the `tom_alerts` library.
+This broker method converts a remote alert into a TOM Toolkit `GenericAlert`.
+
+```python
+@classmethod
+def to_generic_alert(clazz, alert):
+    return GenericAlert(
+        timestamp=alert['timestamp'],
+        url=broker_url,
+        id=alert['id'],
+        name=alert['name'],
+        ra=alert['ra'],
+        dec=alert['dec'],
+        mag=alert['mag'],
+        score=alert['score']
+    )
+```
+In our case, the `GenericAlert` attributes match up *almost* directly with our test
+data. How convenient! We'll just go ahead and define the `GenericAlert`'s `url`
+field as the `broker_url` we retrieved our test data from.
+
+```python
+@classmethod
+def to_generic_alert(clazz, alert):
+  ...
+  url=broker_url,
+  ...
+```
+
+#### `to_target` Class Method
+To convert our new alert into a TOM Target, we'll use `to_target`. Just like
+`to_generic_alert`, we'll simply set the relevant TOM Target parameters to
+the fields from our alert.
+
+```python
+@classmethod
+def to_target(clazz, alert):
+    return Target(
+        identifier=alert['id'],
+        name=alert['name'],
+        type='SIDEREAL',
+        designation='MY ALERT',
+        ra=alert['ra'],
+        dec=alert['dec'],
+    )
+```
+
+## Using Our New Alert Broker
+Now that we've created our TOM alert broker, let's hook it into our TOM
+so that we can ingest alerts and create targets.
+
+The `tom_alerts` module will look in `settings.py` for a list of alert
+broker classes, so we'll need to add `MyBroker` to that list.
+
+```python
+TOM_ALERT_CLASSES = [
+    ...
+    'tom_alerts.brokers.mars.MARSBroker',
+    'mytom.my_broker.MyBroker',
+    ...
+]
+```
+Now, navigate to the top-level directory of your Django project,
+where `manage.py` resides and run
+
+```bash
+./manage.py makemigrations
+./manage.py migrate
+./manage.py runserver
+```
+
+Navigate to [http://127.0.0.1:8000/alerts/query/list/](http://127.0.0.1:8000/alerts/query/list/)
+
+You should now see 'MyBroker' listed as a broker! Clicking the link will bring you
+to the query page, where you can make a query to our sample dataset.
+
+![successful-broker-list]({{"/assets/img/create_broker_doc/success_broker_list.png" | absolute_url}})
```

Comment:
These images don't display for me. But I don't think you need to use a special tag here. 

`![successful-broker-list](/assets/img/create_broker_doc/success_broker_list.png)`

Worked fine for me.